__test__/integration/adapters/collateralized_simple_interest_loan_adapter/scenarios/index.ts

```diff
@@ -0,0 +1,116 @@
+import { CollateralizedSimpleInterestTermsContractContract } from "src/wrappers/contract_wrappers/collateralized_simple_interest_terms_contract_wrapper";
+import { DummyTokenContract } from "src/wrappers/contract_wrappers/dummy_token_wrapper";
+
+// Adapters
+import { SimpleInterestTermsContractParameters } from "src/adapters/simple_interest_loan_adapter";
+import { CollateralizedTermsContractParameters } from "src/adapters/collateralized_simple_interest_loan_adapter";
+
+// Types
+import { DebtOrder } from "src/types/debt_order";
+
+export interface ReturnCollateralScenario {
+    // The test's description.
+    description: string;
+    // True if the scenario succeeds in returning collateral.
+    succeeds: boolean;
+    simpleTerms: SimpleInterestTermsContractParameters;
+    collateralTerms: CollateralizedTermsContractParameters;
+    // A function that returns a valid DebtOrder instance.
+    generateDebtOrder: (
+        debtKernel: DebtKernelContract,
+        repaymentRouter: RepaymentRouterContract,
+        principalToken: DummyTokenContract,
+        termsContract: CollateralizedSimpleInterestTermsContractContract,
+        termsParams: string,
+    ) => DebtOrder.Instance;
+    // Typically the underwriter fills the debt order.
+    orderFiller: string;
+    // True if the debt order's term has lapsed.
+    termLapsed: boolean;
+    // True if the entire debt has been repaid to the creditor.
+    debtRepaid: boolean;
+    // True if the collateral has already been withdrawn.
+    collateralWithdrawn: boolean;
+    // Returns the agreement id that should be provided to the function.
+    givenAgreementId: (agreementId: string) => string;
+    // If there is an error in returning collateral, this defines the expected message.
+    error?: RegExp | string;
+}
+
+let defaultSimpleTerms: SimpleInterestTermsContractParameters;
+let defaultCollateralTerms: CollateralizedTermsContractParameters;
+
+defaultSimpleTerms = {
+    amortizationUnit: "hours",
+    principalAmount: Units.ether(1),
+    interestRate: new BigNumber(0.32),
+    termLength: new BigNumber(1),
+    principalTokenIndex: new BigNumber(0),
+};
+
+defaultCollateralTerms = {
+    collateralTokenIndex: new BigNumber(2),
+    collateralAmount: Units.ether(0.1),
+    gracePeriodInDays: new BigNumber(30),
+};
+
+export function scenarioDefaults(accounts: Array<any>): ReturnCollateralScenario {
+    const debtor = accounts[1];
+    const creditor = accounts[2];
+    const underwriter = accounts[3];
+    const relayer = accounts[4];
+
+    return {
+        description: "default description",
+        succeeds: true,
+        termLapsed: true,
+        debtRepaid: true,
+        orderFiller: underwriter.address,
+        collateralTerms: defaultCollateralTerms,
+        simpleTerms: defaultSimpleTerms,
+        collateralWithdrawn: false,
+        givenAgreementId: (agreementId: string) => agreementId,
+        generateDebtOrder: (
```

Comment:
It doesn't seem like we ever deviate away from this default for `generateDebtOrder` -- why not instead generate this debt order in the Runner?